### PR TITLE
Add Top Line and Deployment Grafana dashboards

### LINKED ISSUE #500: Create deployment grafana dashboard
Create a deployment details dashboard in grafana.  This dashboard should be displayed when a deployment is selected from the deployments directory in the Conduit dashboard.  This grafana dashboard replaces the deployment detail page.  

TODO: what, exactly, does on this dashboard.

### PR DESCRIPTION
**Pull request by siggy**
Existing Grafana configuration contained no dashboards, just a skeleton
for testing.

Introduce two Grafana dashboards:
1) Top Line: Overall health of all Conduit-enabled services
2) Deployment: Health of a specific conduit-enabled deployment

Fixes #500

Signed-off-by: Andrew Seigner <siggy@buoyant.io>

## Top Line

screen shot 2018-03-12 at 4 48 27 pm

## Deployment

screen shot 2018-03-12 at 5 17 57 pm

screen shot 2018-03-12 at 5 18 40 pm

### CODE REVIEW

**Comment by dadjeibaah** [comment]
I am not entirely familiar of what the requirements are for the grafana integration. Are we going to be adding dashboards for service mesh metrics? i.e. `destination`, tap` and other control plane pods?

File: cli/install/viz.go [hunk 1]
```diff
@@ -2,66 +2,925 @@ package install
...
+      "editable": true,
+      "gnetId": null,
+      "graphTooltip": 1,
+      "id": null,
+      "iteration": 1520894444409,
+      "links": [],
+      "panels": [
+        {
+          "content": "<div>\n  <div style=\"position: absolute; top: 0, left: 0\">\n    <a href=\"https://conduit.io\" target=\"_blank\"><img src=\"https://conduit.io/images/conduit-primary-white.svg\" style=\"height: 30px;\"></a>\n  </div>\n  <div style=\"position: absolute; top: 0; right: 0; font-size: 15px\">\n    <a href=\"https://conduit.io\" target=\"_blank\">Conduit</a> is a next-generation ultralight service mesh for Kubernetes.\n    <br>\n    Need help? Visit <a href=\"https://conduit.io\" target=\"_blank\">conduit.io</a>.\n  </div>\n</div>",
+          "gridPos": {
```
Comment (dadjeibaah) [inline cli/install/viz.go]: Is there any way we can wrap this line. I am thinking it might be difficult because it is a template string.

**Comment by siggy** [comment]
@dadjeibaah Re: service mesh metrics, you're correct, we have a plan add a `conduit health` dashboard, which will expose metrics specific to the health of the control plane pods. that task is captured in #420, and is also dependent on #583, which sets up a new Prometheus job called `conduit-controller` to collect exactly these service mesh metrics.

File: cli/install/viz.go (hunk 1 above)
Comment (siggy) [reply cli/install/viz.go]: Good question. I agree wrapping these would make it a lot more readable. Unfortunately the contents of these files is an exact copy/paste of Grafana's rendering of dashboard JSON. The process of editing a dashboard right now is:
1) Install conduit
2) Edit dashboard in Grafana web UI
3) Export JSON
4) Paste JSON into `viz.go`

We may want to automate this process at some point, which would then allow us to do formatting things like your suggestion.

**Comment by adleong** [comment]
Will this PR address how to get these dashboards into the Conduit dashboard?  Grafana is accessable at `http://127.0.0.1:XXX/api/v1/namespaces/conduit/services/grafana:http/proxy` if the `conduit dashboard` proxy is running but doesn't load assets correctly because `base_url` is not set in the grafana config.  

I think if we set base_url correctly, it should be possible to directly link to the deployment grafana dashboard from the web UI.

**Comment by siggy** [comment]
@adleong Re: dashboard access, there's an `Enable user access to the Grafana dashboard` task listed in #420 to solve this, via some combination of `base_url`, `conduit dashboard`, or other method.

**Pull request merged**